jsep.go

```diff
@@ -51,11 +51,15 @@ var extMapURI = map[int]string{
 // NewJSEPSessionDescription creates a new SessionDescription with
 // some settings that are required by the JSEP spec.
 func NewJSEPSessionDescription(identity bool) *SessionDescription {
```

Comment:
It must break API to return `(*SessionDescription, error)` to use `crypto/rand` in SessionID without panic.
@Sean-Der what do you think?

Reply:
100% support from me!

I would be fine even doing a new minor version. It isn't a big change, and there is an API difference so users will clearly see.